blocks/feature-layout/feature-layout.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Loads and decorates the feature-layout block.
+ * @function
+ * @param {Element} block the block element
+ * @returns {Element}
+ */
+
+export default async function decorate (block) {
+  const backgroundColor = block.children[0]?.innerText?.trim();
```

Comment:
can we use object syntax? it's an easy way to see that `data.backgroundColor` comes from content entry but like `gridContainer` is something generated in the file.